app/assets/stylesheets/items.scss

```diff
@@ -0,0 +1,59 @@
+.jumbotron {
+  background-color: #94618E !important;
```

Comment:
Don't use important: https://css-tricks.com/when-using-important-is-the-right-choice/